src/main/java/uk/gov/justice/hmpps/casenotes/dto/CaseNote.java

```diff
@@ -76,8 +76,76 @@
     @ApiModelProperty(value = "Location Id representing where Case Note was made.", position = 14, example = "MDI")
     private String locationId;
 
+    @ApiModelProperty(required = true, value = "Delius number representation of the case note id - will be negative for sensitive case note types", position = 15, example = "-23")
+    @NotNull
+    private Integer eventId;
```

Comment:
need `eventId` so poll pusher can grab an integer representation of the case note id to push to delius